pkg/util/intstr/intstr.go

```diff
@@ -101,6 +105,17 @@ func (intstr IntOrString) MarshalJSON() ([]byte, error) {
 	}
 }
 
+func (_ IntOrString) OpenAPIDefinition() common.OpenAPIDefinition {
```

Comment:
I see your point, but at the same time intOrString is effectively part of our API. Maybe we should split it into two types, and move a copy into our api somewhere? (My preference is to nuke IntOrString from orbit but I think that's not on the table at the moment :) )